packages/strapi-plugin-upload/admin/src/components/InputMedia/EmptyInputMedia.js

```diff
@@ -10,10 +15,44 @@ const EmptyInputMedia = styled.div`
   justify-content: center;
   align-items: center;
   border-radius: ${({ theme }) => theme.main.sizes.borderRadius};
-  background-color: ${({ theme }) => theme.main.colors.black};
+  background-color: ${({ theme, isInDropZone }) =>
+    isInDropZone ? '#515764' : theme.main.colors.black};
   cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
+
+  & * {
+    pointer-events: none;
+  }
```

Comment:
ℹ️ CSS hack to prevent the dragenter / dragleave event to trigger when browsing over a children node (like the text content or the background svg icon)